app/main.py

```diff
@@ -65,14 +59,9 @@ def upload():
     empty_directory()
     csv = request.form['csv'].strip()
     img = request.form['img-default']
-    eventyay_url = request.form['eventyay_url'].strip()
     file = request.files['file']
-    _pdf = True if request.form.get('pdf') == 'on' else False
-    _zip = True if request.form.get('zip') == 'on' else False
-
-    if not _pdf and not _zip:
-        flash('Please select a download filetype!', 'error')
-        return redirect(url_for('index'))
+    _pdf = True  if request.form.get("pdf") == 'on' else False
```

Comment:
Since there is no radio button for pdf and/or zip anymore in your changes, therefore this line can be removed.
And any subsequent changes related to it must be dealt with. Now your PR only has a Submit button therefore it can be removed and appropriate changes must be made to `main.py`

Reply:
@ParthS007 Please fix this